test/projectManager.ts

```diff
@@ -166,6 +166,36 @@ export class ProjectManager {
             return ProjectManager.runPlatform(testRunDirectory, targetPlatform, true, targetEmulator);
         }
     }
+    
+    /**
+     * Navigates away from the application and then navigates back to it.
+     */
+    public static resumeApplication(delayBeforeResumingMs: number, targetPlatform: platform.IPlatform, namespace: string, testRunDirectory: string, targetEmulator: string): Q.Promise<void> {
+        var emulatorManager = targetPlatform.getOptionalEmulatorManager();
```

Comment:
Yea I agree with you, I was following the file's convention, but moving it into IEmulatorManager is better.